### DISCUSSION TITLE
disable-zoom not working

### DISCUSSION THREAD

**Discussion by lloyd-thomas** [google/model-viewer#1935] [Q&A]
### Description

I have added the attribute disable-zoom but it doesn't seem to be working for me.

#### Live Demo
https://domh.net/model-viewer/

### Version
https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js

### Browser Affected
All

### OS
All

It seems to be working in the example section of the model-viewer site here: https://modelviewer.dev/examples/stagingandcameras/#disableZoom

**Comment by elalish** [comment]
This feature was just added, so it'll be available with our next release, v1.4.

**Comment by lloyd-thomas** [comment]
Thank you for getting back so quickly, and thank you for such an amazing component. When will the next release be available? and for the time being are there any workarounds?

**Comment by elalish** [comment]
@lloyd-thomas Well, you're always welcome to build from source and deploy it yourself. We generally release on the same cadence as Chrome (every 6 weeks or so) and 87 just rolled out so we're due. Probably early December.

**Comment by lloyd-thomas** [comment]
@elalish thanks again for the info. Building from source now that's an idea, what's the best way of doing that?

**Comment by elalish** [comment]
Complete instructions are in the README. Might be a little trickier if you're on Windows. I hear WSL works, but you need to check that your node is up to date.

**Comment by lloyd-thomas** [comment]
Yeah, I'm on MacOS I did manage to download the zip and get the dev environment running locally, I ran the build but when transferring the js files to my site I was getting a bunch of errors even outside my webpack setup. I gave up eventually, and removed the site/app as it was taking up too much space on my drive. Now I'm eagerly waiting the next release. Thanks for looking out for me.

**Comment by pecollinni7** [comment]
@elalish hello there
any news on when will the 1.4 version be released? im also in the need for the disable-zoom attribute :)
cheers

**Comment by sumasamus** [comment]
Hi, I have seen that in the code on the example page, It is working ... Anyone know why?

https://modelviewer.dev/examples/stagingandcameras/index.html#disableZoom

![Captura](https://user-images.githubusercontent.com/76538109/103039287-c80ed100-4570-11eb-8701-a6ad5c4e2db7.PNG)

Thanks in any case.

**Comment by elalish** [comment]
@sumasamus See my first comment. The docs are served from master, not the latest release, hence why it is testable.
